etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/orders/Order.java

```diff
@@ -11,4 +11,14 @@ public interface Order<T> {
     String getFhirResourceId();
 
     String getPatientId();
+
+    String getPlacerOrderNumber();
+
+    String getSendingApplicationDetails();
+
+    String getSendingFacilityDetails();
+
+    String getReceivingApplicationDetails();
+
+    String getReceivingFacilityId();
```

Comment:
For consistency, should we have `getReceivingFacilityDetails` instead of `getReceivingFacilityId`?

Reply:
As an idea, instead of `String` we could return a custom object that will split namespace, id and type

Reply:
Yes. We haven't started working on this method. Initially, we had  the methods return single IDs, now they are returning a string with multiple IDs. Yes, we are looking at the possibility of having a pojo and using fhir path as well.